lib/parse_backup_json.js

```diff
@@ -1,22 +1,30 @@
 const fs = require('fs');
+const Transform = require('stream').Transform;
 
-const readFilePromise = fileName => new Promise((res, rej) => {
-  fs.readFile(fileName, 'utf8', (err, resp) => err? rej(err): res(resp));
-})
+const makePrettyJsonFile = (inputFileName, outputFileName) => {
+  let isBegin = true;
+  const parser = new Transform({
+    transform : function (data, _, done) {
```

Comment:
arrow function doesn't work cause I use this.push and binding doesn't work properly :(